catalog-rest-service/src/main/resources/json/schema/entity/policies/accessControl/rule.json

```diff
@@ -11,24 +11,27 @@
       "description": "This schema defines all possible operations on metadata of data entities.",
       "type": "string",
       "enum": [
-        "SuggestDescription",
-        "SuggestTags",
-        "UpdateDescription",
-        "UpdateOwner",
-        "UpdateTags",
-        "UpdateLineage",
-        "DecryptTokens",
-        "UpdateTeam"
-      ],
-      "javaEnums": [
-        { "name": "SuggestDescription" },
-        { "name": "SuggestTags" },
-        { "name": "UpdateDescription" },
-        { "name": "UpdateOwner" },
-        { "name": "UpdateTags" },
-        { "name": "UpdateLineage" },
-        { "name": "DecryptTokens" },
-        { "name": "UpdateTeam" }
+        "Create",
```

Comment:
Observation:

The previous naming convention was `{action;required}{target-object-or-attribute;required}`

The new one seems to be `{target-object;optional}{action:required}{target-attribute:optional}`

Reply:
Except ViewAll and EditAll perhaps (where I'm not sure whether All refers to entities or attributes.